recipe/meta.yaml

```diff
@@ -11,27 +11,41 @@ source:
     - 0006-Add-support-for-ppc64le-embedded-jdk.patch 
 
 build:
-  number: 3
+  number: 4
   ignore_prefix_files: True
   binary_relocation: False  # [osx]
 
+ignore_run_exports:
```

Comment:
`ignore_run_exports` is part of the `build` stanza, so this needs an indent.

Reply:
Yes.